frontends/web/src/utils/event.ts

```diff
@@ -60,11 +42,14 @@ const subscriptions: Subscriptions = {};
 /**
  * This function dispatches the events from the websocket to the observers.
  */
-function handleEvent(payload: any): void {
-  if (typeof payload.subject === 'string') {
+function handleEvent(payload: TPayload): void {
```

Comment:
not `TEvent`?

Reply:
nope, same as other comment, after the checks TypeScript knows it is TEvent.

![Screen Shot 2022-10-19 at 15 05 54](https://user-images.githubusercontent.com/546900/196699228-59c2aa87-1e74-470e-aca6-7c096272158d.png)